iOS_SDK/OneSignalSDK/Source/OSNotification.m

```diff
@@ -187,7 +273,51 @@ -(void)parseOtherApnsFields {
 }
 
 - (NSString *)description {
-    return [NSString stringWithFormat: @"notificationID=%@ templateID=%@ templateName=%@ contentAvailable=%@ mutableContent=%@ category=%@ rawPayload=%@", _notificationID, _templateID, _templateName, _contentAvailable ? @"YES" : @"NO", _mutableContent ? @"YES" : @"NO", _category, _rawPayload];
+    return [NSString stringWithFormat: @"notificationID=%@ templateID=%@ templateName=%@ contentAvailable=%@ mutableContent=%@ category=%@ rawPayload=%@", _notificationId, _templateId, _templateName, _contentAvailable ? @"YES" : @"NO", _mutableContent ? @"YES" : @"NO", _category, _rawPayload];
+}
+
+
+- (NSString*)stringify {
+    NSMutableDictionary *obj = [NSMutableDictionary new];
+    [obj setObject:[NSMutableDictionary new] forKeyedSubscript:@"payload"];
+    
+    if (self.notificationId)
+        [obj[@"payload"] setObject:self.notificationId forKeyedSubscript: @"notificationID"];
```

Comment:
`notificationID` should be `notificationId`

Reply:
fixed!